code/modules/antagonists/traitor/objectives/kidnapping.dm

```diff
@@ -309,8 +310,9 @@
 	sent_mob.flash_act()
 	sent_mob.adjust_confusion(10 SECONDS)
 	sent_mob.adjust_dizzy(10 SECONDS)
-	sent_mob.set_eye_blur_if_lower(100 SECONDS)
+	sent_mob.set_eye_blur_if_lower(50 SECONDS)
 	sent_mob.dna.species.give_important_for_life(sent_mob) // so plasmamen do not get left for dead
+	to_chat(sent_mob, span_hypnophrase(span_reallybig("You can't remember a thing leading up to the kidnapping, and your head hurts like hell...")))
```

Comment:
I personally don't like this sort of thing because it's placing annoying restrictions on the kidnapped player to act like nothing has happened.
If players don't want to get discovered during kidnapping, then they should disguise themselves and make themselves harder to identify.
Please remove.

Reply:
That's the main issue with kidnapping, though. It's extremely difficult to hide one's identity, and doing so as a nonhuman (like a lizard or moth) requires modsuits or bodycoverings with extreme slowdown. The reason the metaprotection is in place for heretics is the exact same reason.

Reply:
Heretics don't have as much gear to hide their identity as Traitors do

Reply:
Also kidnappings are not mandatory to progress as a traitor unlike heretics.